### ISSUE TITLE
[BUG] This fix probably doesn't do anything

### ISSUE THREAD

**Issue by wywywywy** [Curlypla/peft-4bit-fix#1]
We've run some comparisons, and figured out that this fix probably isn't working as intended.

Please see this issue: https://github.com/oobabooga/text-generation-webui/issues/332

To reproduce, fix the generation to a specific seed, then generate with and without LoRA. The results unfortunately are the same, which means LoRA isn't applied.

Below is from seed 12, LLaMa 13b weights with alpaca13B-lora.

Prompt
```
Below is an instruction that describes a task.
Write a response that appropriately completes the request.

### Instruction:
In 10 sentences, summarise the seminal paper called "Attention is All You Need" by Vaswani et al. in 2017
### Response:
```

Parameters
```
do_sample=True
temperature=0.36
top_p=1
typical_p=1
repetition_penalty=1.23
top_k=12
num_beams=1
penalty_alpha=0
min_length=0
length_penalty=1
no_repeat_ngram_size=0
early_stopping=False
```

Output

> Below is an instruction that describes a task.
> Write a response that appropriately completes the request.
> 
> ### Instruction:
> In 10 sentences, summarise the seminal paper called "Attention is All You Need" by Vaswani et al. in 2017
> ### Response::
> The authors of this article proposed a new model for attention called Attention Is All You Need (AITAN). The AITAN model has three components: a multi-headed encoder and decoder with self-attentions to capture long range dependencies across input sequences; a softmax classifier which predicts labels from hidden states; and a single linear layer on top of the output of the last head to compute contextualized representations of tokens. They also used the transformer architecture as their base network because it can be trained efficiently using gradient descent methods such as Adam or stochastic gradient decent. In addition, they have shown how to use the Transformer architecture for sequence tagging tasks like named entity recognition and part-of-speech tagging. Finally, the authors have demonstrated that their approach outperforms previous state-of-the art models for these two tasks.
